packages/Combobox/src/Combobox.stories.tsx

```diff
@@ -0,0 +1,261 @@
+import React from 'react';
+
+import { ComponentMeta, ComponentStory } from '@storybook/react';
+import { within, userEvent } from '@storybook/testing-library';
+import { expect } from '@storybook/jest';
+
+import Tag from '@igloo-ui/tag';
+import Happiness from '@igloo-ui/icons/dist/Happiness';
+import { OptionType } from '@igloo-ui/list';
+
+import Section from '@components/section';
+import readme from '../../Combobox/README.md';
+
+import Combobox, { ComboboxOption } from './Combobox';
+
+export default {
+  title: 'Components/Combobox',
+  component: Combobox,
+  parameters: {
+    description: readme,
+  },
+  decorators: [
+    (Story) => (
+      <div
+        style={{
+          minHeight: '34rem',
+        }}
+      >
+        <Story />
+      </div>
+    ),
+  ],
+} as ComponentMeta<typeof Combobox>;
+
+const comboboxPlaceholder = 'ex: Lorem ipsum dolor';
+
+const handleOnChange = (option: OptionType | undefined): void => {
+  const item = option as ComboboxOption;
+  alert(`Option: ${item?.label}`);
+};
+
+const smallOptionList: ComboboxOption[] = [
+  {
+    label: 'Text option',
+    value: 'text',
+  },
+  {
+    label: 'Disabled option',
+    value: 'disabled',
+    disabled: true,
+  },
+  {
+    label: 'Text option with icon',
+    value: 'icon',
+    icon: <Happiness size="small" />,
+  },
+];
+
+const largeOptionList: ComboboxOption[] = [
+  {
+    label: 'Text 1',
+    value: '1',
+  },
+  {
+    label: 'Text 2 (disabled)',
+    value: '2',
+    disabled: true,
+  },
+  {
+    label:
+      'Text 3. I will put a lot of text here to see how it behaves. Hopefully it looks good!',
+    value: '3',
+    color: '#74DCC9',
+  },
+  {
+    label: 'Text 4',
+    value: '4',
+  },
+  {
+    label: 'Text 5',
+    value: '5',
+  },
+  {
+    label: 'Text 6',
+    value: '6',
+  },
+];
+
+const Template: ComponentStory<typeof Combobox> = (args) => (
+  <Combobox {...args} />
+);
+export const Overview = Template.bind({});
+Overview.args = {
+  children: comboboxPlaceholder,
+  options: smallOptionList,
+};
+Overview.play = async ({ canvasElement }) => {
+  const canvas = within(canvasElement);
+
+  await userEvent.click(canvas.getByRole('button'));
+  const firstOption = await canvas.findByText('Text option');
+
+  await expect(firstOption).toBeInTheDocument();
+};
+
+export const Sizes = () => (
+  <Section column>
+    <Combobox options={smallOptionList} onChange={handleOnChange}>
+      Default
+    </Combobox>
+    <Combobox
+      options={smallOptionList}
+      onChange={handleOnChange}
+      isCompact={true}
+    >
+      Compact
+    </Combobox>
+  </Section>
+);
+
+export const States = () => (
+  <Section column>
+    <Combobox
```

Comment:
We should not see the state of focus in disabled

<img width="986" alt="disabled-focus" src="https://user-images.githubusercontent.com/2379339/218614329-59cb1c50-d795-47f9-8954-5ed8075d1dcb.png">